app/templates/secrets.hbs

```diff
@@ -9,7 +9,7 @@
         <span class="icon icon-menu">menu</span>
       {{/link-to}}
       <form action="#">
-        {{input placeholder="Search..." value=query}}
+        {{input id="search" placeholder="Search..." value=query}}
```

Comment:
The {{ input }} ember component already defines an id, which you are overriding. I'm not sure if this a good idea.